docs/api/amqp.md

```diff
@@ -172,6 +173,33 @@ JSON in the following format:
 }
 ```
 
+
+### Device removed
+
+Message with the device unregistered on the cloud.
+
+#### Exchange
+
+* `fog`
+
+#### Binding Key
+
+* `device.unregistered`
+
+#### Expected Response
+
+JSON in the following format:
+  * `device` **Object** devices representation
```

Comment:
Please, remove this line because the JSON only contains `id` property.

Reply:
Done